VirtoCommerce.Storefront.Tests/Features/Samples/full_data.json

```diff
@@ -0,0 +1,156 @@
+{
+  "features": [
+    {
+      "Name": "OrdersBrowsing",
+      "IsActive": true,
+      "Requires": [],
+      "Replaces": [],
+      "Replaced": [],
```

Comment:
Why are these two parameters necessary? Is this not redundant? Two sources of truth.

Reply:
Agree. The replaced property was removed